paging/core/build.gradle.kts

```diff
@@ -57,27 +44,11 @@ kotlin {
 }
 
 android {
-    namespace = "org.mobilenativefoundation.store.paging5"
+    namespace = "org.mobilenativefoundation.paging.core"
```

Comment:
you can also do org.mobilenativefoundation.store5.paging.core

in case you want to ever have something like 5/6 at same time